includes/user.php

```diff
@@ -447,7 +447,7 @@ function ap_user_top_posts_tab(){
     
     $link = '?tab=';
     ?>
-    <?php printf(__('Top %s', 'ap'), $active); ?>
+    <?php printf(__('Top %s', 'ap'), __(ucfirst($active), 'ap')); ?>
```

Comment:
@cyress This will never work, cause you are using PHP variable inside gettext function. And also upper case will not work if language not English.